### PR TITLE
liz - sort users table by default

### LINKED ISSUE #15: FEATURE: Admin table should not move entries after a toggle button is pressed
### As a...

admin

### I can...

toggle a user's admin, driver, or rider without moving the entries in the table

### So that...

the user's table stays consistent

### Discussion

Here is what is currently happening:

User table before toggling "Driver" for Aasish Virjala:

Screenshot 2024-05-15 at 5 23 44 PM

User table after toggling "Driver" for Aasish Virjala:

Screenshot 2024-05-15 at 5 23 51 PM

This needs to be fixed so that the entries in the table stay consistent.

### Acceptance Criteria

- [ ] User table is sorted by some metric (you decide...)
- [ ] When a property of a user is toggled, they should remain in the same position in the table (instead of moving to the bottom)

### Implementation Todos

Frontend:
- [ ] manipulate the table so that it stays sorted by some sort of metric (maybe id?)

Testing:
- [ ] Front end tests pass and there is adequate coverage

### PR DESCRIPTION
**Pull request by lizzyinnie**
Closes #15 

Automatically sorts the UsersTable by <last name, first name, id> so that when a user is updated in the db, the table stays in a predictable order.

Modified the users fixtures to demonstrate this, and adjusted the tests accordingly.

Dokku deployment: [here](https://proj-gauchoride-andyinnie-dev.dokku-14.cs.ucsb.edu/)

### CODE REVIEW

**Review by lindseywn — DISMISSED** [review]
LGTM, works properly according to acceptance criteria, tests pass as well

File: frontend/src/main/components/Users/UsersTable.js [hunk 1]
```diff
@@ -112,6 +112,16 @@ export default function UsersTable({ users}) {
 
     //const columnsToDisplay = showButtons ? buttonColumn : columns;
```
Comment (pconrad) [inline frontend/src/main/components/Users/UsersTable.js]: Can we remove this comment out line? 

I know it isn't from you, but when we see these in files we touch, it's good to get rid of them.

Comment (pconrad) [reply frontend/src/main/components/Users/UsersTable.js]: See also: 

https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed

**Review by pconrad — CHANGES_REQUESTED** [review]
Tested on dokku and works as promised; code is good!

Just one thing.. could we remove that one commented out line (even though it isn't from you?)

**Review by diyaparm — COMMENTED** [review]
LGTM removed comments that was requested.

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**